src/test/java/worktypeclassic/CreateWorkTypeTest.java

```diff
@@ -2,20 +2,22 @@
 
 import base.BaseTest;
 import org.testng.annotations.Test;
+import salesforce.PageTransporter;
 import salesforce.classic.WorkTypeClassicPage;
 import salesforce.classic.WorkTypeForm;
 import salesforce.classic.WorkTypeInfo;
+
 import static org.testng.Assert.assertEquals;
 
 public class CreateWorkTypeTest extends BaseTest {
 
     public void createWorkTypeToClassicSalesforce() {
-        WorkTypeClassicPage workType = loginPage.goToClassic();
```

Comment:
maybe you could set the login part in a before test, since is something you will use for other test.

Reply:
solved